salt/core/ntp.sls

```diff
@@ -0,0 +1,23 @@
+# Disable pre-installed systemd-timesyncd as we prefer NTP
+systemd-timesyncd:
+  service.dead:
+    - enable: False
+
+ntp:
+  pkg:
+    - installed
+  service.running:
+    - enable: True
+    - watch: # restart if we change the config
+      - file: /etc/ntp.conf
```

Comment:
For fail2ban and ssh, we `reload: True`. Does ntp support reload? Or do we definitely want to restart?

Reply:
NTP doesn't support reloads. It restarts really quickly so restarting isn't a problem.